packages/store/tests/select.spec.ts

```diff
@@ -68,250 +66,6 @@ describe('Select', () => {
 
   const states = [MySubState, MySubSubState, MyState];
 
-  it('should throw an exception when the user has forgotten to import the NGXS module', () => {
```

Comment:
We should keep these tests for `@Select` until the decorator is actually removed. We might need to add a linting hint to ignore the deprecation notice here.